packages/editor/src/components/widgets/accordion/Accordion.tsx

```diff
@@ -32,15 +32,18 @@ type AccordionHeaderProps = {
   title?: string;
   control?: { label: string; icon: IvyIcons; action: () => void };
   children: ReactNode;
+  isAccordionOpen: boolean;
 };
 
-export const AccordionHeader = ({ title, control, children }: AccordionHeaderProps) => (
+export const AccordionHeader = ({ title, control, children, isAccordionOpen }: AccordionHeaderProps) => (
```

Comment:
we are in the accordion, so imo no needed to duplicate this here -> rename to isOpen